lib/_tls_wrap.js

```diff
@@ -584,6 +584,17 @@ TLSSocket.prototype._start = function() {
   this._handle.start();
 };
 
+TLSSocket.prototype._isSessionResumed = function _isSessionResumed(session) {
+  if (!session)
+    return false;
+
+  var next = this.getSession();
+  if (!next)
+    return false;
+
+  return Buffer.compare(session, next) === 0;
```

Comment:
don't care if you make this change, but could use `return session.equals(next);` (assuming they're both buffers).